nodeup/pkg/model/kube_apiserver.go

```diff
@@ -732,6 +732,7 @@ func (b *KubeAPIServerBuilder) buildPod(kubeAPIServer *kops.KubeAPIServerConfig)
 
 func (b *KubeAPIServerBuilder) buildAnnotations() map[string]string {
 	annotations := make(map[string]string)
+	annotations["kubectl.kubernetes.io/default-logs-container"] = "kube-apiserver"
```

Comment:
[This KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-cli/2227-kubectl-default-container/README.md) is moving to default-container to cover exec and other kubectl commands.

```suggestion
	annotations["kubectl.kubernetes.io/default-container"] = "kube-apiserver"
```